src/slider/slider.js

```diff
@@ -30,6 +30,7 @@
     this.element_ = element;
     // Browser feature detection.
     this.isIE_ = window.navigator.msPointerEnabled;
+    this.isEdge_ = !this.isIE_ && !!window.StyleMedia;
```

Comment:
The problem comes from an IE only CSS rule that sets the height of the .mdl-slider element to 32px, Edge also needs this fix.
I modified the selector to include Edge but then noticed MaterialSlider.init was doing browser detection as well (to create DOM elements for the missing CSS hooks in Chrome).

What can I do now? is the current fix still unacceptable?